### PR TITLE
Remove monkeypatching for ES7.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Bump libs

### CODE REVIEW

File: requirements/default.txt [hunk 1]
```diff
@@ -283,16 +296,16 @@ djangorestframework==3.11.2 \
     --hash=sha256:5cc724dc4b076463497837269107e1995b1fbc917468d1b92d188fd1af9ea789 \
     --hash=sha256:a5967b68a04e0d97d10f4df228e30f5a2d82ba63b9d03e1759f84993b7bf1b53
     # via -r requirements/default.in
-elasticsearch==1.2.0 \
-    --hash=sha256:53a822b01b0494df9658c04e87216180e5ecbe85b178673dd848b1f809d5c100 \
-    --hash=sha256:bb93d85ca7e50d68e4197852bb6dba6098bb772500f82f5db2ff4a80711505c9
+elasticsearch-dsl==7.3.0 \
```
Comment (akatsoulas) [inline requirements/default.txt:299]: This feels nice :)

File: requirements/default.txt [hunk 2]
```diff
@@ -368,9 +381,9 @@ idna==2.10 \
     # via
     #   -r requirements/default.in
     #   requests
-jinja2==2.11.3 \
-    --hash=sha256:03e47ad063331dd6a3f04a43eddca8a966a26ba0c5b7207a9a9e4e08f1b29419 \
-    --hash=sha256:a6d58433de0ae800347cab1fa3043cebbabe8baa9d29e668f1c768cb87a333c6
+jinja2==3.0.1 \
```
Comment (akatsoulas) [inline requirements/default.txt:384]: @LeoMcA working on the deprecation per your comment on the last PR

**Review by LeoMcA — APPROVED** [review]
r+wc

Do we need to update the ADRs to remove references to search v2, since that was the terminology we were using at the time?

I haven't gone through the requirements updates, I'm thinking it might be worth resetting the changes and doing `pip-compile-multi -g default --no-upgrade` in this PR. We can then update all our libs, and make the Jinja2 changes (and any others required), in a separate PR.

**Comment by lawpac10** [comment]
I'm not the one demanding for any changes in my GitHub  account . Kindly
disregard any request asking you to make deletion or changes in my account
. I guess my account was compromised by a thief . Disregard any attempt to
remove or change any setting in my account .thanks

**Comment by andrewhager9** [comment]
Help me get rid of this hacker

**Comment by lawpac10** [comment]
Please kindly disregard all the request asking for changings in my account
. I never ask for change .the criminals are working against me .don't let
them succeed .

**Comment by andrewhager9** [comment]
???

File: kitsune/search/v2/__init__.py [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-
-# Monkey patch PYTHONPATH to temporarily work with multiple
-# versions of elasticsearch
-sys.path.insert(0, "/vendor")
-
-# Manually import elasticsearch v7.x
-f, filename, description = imp.find_module("elasticsearch")
-elasticsearch7 = imp.load_module("elasticsearch7", f, filename, description)
-
-default_app_config = "kitsune.search.apps.SearchV2Config"
```
[the remark this answers is not in the document]
Comment (akatsoulas) [reply kitsune/search/v2/__init__.py:12]: good catch! I am pretty sure that I had it in there but with a couple of resets, lost it along the way

File: kitsune/search/models.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import logging
```
[the remark this answers is not in the document]
Comment (akatsoulas) [reply kitsune/search/models.py:1]: After the migrations are run we can drop it completely. Right now the classed defined in this file are referenced in the migrations and we do have a pending one that will drop the tables.

**Pull request merged**